revenuecat_examples/purchase_tester/analysis_options.yaml

```diff
@@ -0,0 +1,6 @@
+include: package:pedantic/analysis_options.1.8.0.yaml
+analyzer:
+  exclude:
+    # Ignore generated files
+    - '**/*.g.dart'
+    - 'lib/src/generated/*.dart'
```

Comment:
Was this also causing issues?

Reply:
Ok, I saw how this was causing issues in this other PR https://github.com/RevenueCat/purchases-flutter/pull/285